tests/eth_rpc_api_functions_test.go

```diff
@@ -196,3 +197,58 @@ func makeTestEngine(gdb *gossip.Store) (*abft.Lachesis, *vecmt.Index) {
 	engine := abft.NewLachesis(cdb, nil, nil, nil, abft.LiteConfig())
 	return engine, vecClock
 }
+
+func TestEthConfig_ProducesReadableConfig(t *testing.T) {
+
+	session := getIntegrationTestNetSession(t, opera.GetBrioUpgrades())
+
+	client, err := session.GetClient()
+	require.NoError(t, err)
+	defer client.Close()
+
+	response := map[string]map[string]any{}
+	// var response configResponse
+	err = client.Client().Call(&response, "eth_config")
+	require.NoError(t, err, "eth_config failed")
+	response["current"]["activationTime"] = uint64(response["current"]["activationTime"].(float64))
+
+	// get time from block one because genesis block has time zero
+	// and upgrades gets updated in block 1
+	block1, err := client.BlockByNumber(t.Context(), big.NewInt(1))
+	require.NoError(t, err, "could not get block 1 to determine expected ActivationTime")
+	activationTime := block1.Header().Time
+
+	want := map[string]map[string]any{
+		"current": {
+			"activationTime": activationTime,
+			"blobSchedule":   nil,
+			"chainId":        "0xfa3",
+			"forkId":         "0xcb291288",
+			"precompiles": map[string]any{
+				"BLAKE2F":              "0x0000000000000000000000000000000000000009",
+				"BN254_ADD":            "0x0000000000000000000000000000000000000006",
+				"BN254_MUL":            "0x0000000000000000000000000000000000000007",
+				"BN254_PAIRING":        "0x0000000000000000000000000000000000000008",
+				"ECREC":                "0x0000000000000000000000000000000000000001",
+				"ID":                   "0x0000000000000000000000000000000000000004",
+				"KZG_POINT_EVALUATION": "0x000000000000000000000000000000000000000a",
+				"MODEXP":               "0x0000000000000000000000000000000000000005",
+				"RIPEMD160":            "0x0000000000000000000000000000000000000003",
+				"SHA256":               "0x0000000000000000000000000000000000000002",
+				"BLS12_G1ADD":          "0x000000000000000000000000000000000000000b",
+				"BLS12_G1MSM":          "0x000000000000000000000000000000000000000c",
+				"BLS12_G2ADD":          "0x000000000000000000000000000000000000000d",
+				"BLS12_G2MSM":          "0x000000000000000000000000000000000000000e",
+				"BLS12_MAP_FP2_TO_G2":  "0x0000000000000000000000000000000000000011",
+				"BLS12_MAP_FP_TO_G1":   "0x0000000000000000000000000000000000000010",
+				"BLS12_PAIRING_CHECK":  "0x000000000000000000000000000000000000000f",
+				"P256VERIFY":           "0x0000000000000000000000000000000000000100",
+			},
+			"systemContracts": map[string]any{"HISTORY_STORAGE": strings.ToLower(fmt.Sprintf("%s", params.HistoryStorageAddress))},
+		},
+		"next": nil,
+		"last": nil,
```

Comment:
Can you add a network rule change to check that the "last" field is filled correctly?